client/my-sites/migrate/migrate-button.jsx

```diff
@@ -31,22 +31,24 @@ class MigrateButton extends Component {
 
 		const message = (
 			<>
-				<h1>
-					{ translate( 'Import and overwrite everything on %(targetDomain)s?', {
+				<h1>{ translate( 'Import and replace everything on this site?' ) }</h1>
+				<div>
+					{ translate( 'All posts, pages, comments and media will be lost on %(targetDomain)s.', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'Overwrite %(targetDomain)s? All posts, pages, comments and media will be lost on this WordPress.com site.' )` *ES Score: 7*

Reply:
Changed the wording.